gw_eccentricity/load_data.py

```diff
@@ -712,9 +724,12 @@ def load_lvcnr_waveform(**kwargs):
 def load_sxs_catalogformat(**kwargs):
     """Load modes from sxs waveform files in sxs catalog format.
 
-    This function is intended for loading waveform modes from files in
-    the sxs catalog format (see
+    This function is intended for loading waveform modes from files in the sxs
+    catalog in the new format, i.e., from 2023 onward (see
     https://data.black-holes.org/waveforms/documentation.html).
+
+    For loading sxs catalog waveforms in old format see
+    `load_sxs_catalogformat_old`.
```

Comment:
Be more helpful: say old means rhOhverM..., new means Strain..

Reply:
Done